appengine/gallery/template.soy

```diff
@@ -24,23 +24,14 @@
  * Web page structure.
  */
 {template .start}
-  <table width="100%">
-    <tr>
-      <td>
-        <h1>
-          {call BlocklyGames.soy.titleSpan}
-            {param appName}
-              {$ij.appName}
-              {{msg meaning="Gallery" desc="Display of art created by user's programs.\n{lb}{lb}Identical|Gallery{rb}{rb}"}}Gallery{{/msg}}
-            {/param}
-          {/call}
-        </h1>
-      </td>
-      <td class="farSide">
-        <select id="languageMenu"></select>
-      </td>
-    </tr>
-  </table>
+  {call .messages /}
```

Comment:
Is a call to .messages needed here?

More importantly, is .messages needed at all?  Looks like dead code.

Reply:
Looks like it. I'll remove the Movie.submitDisabled here, its defined in Movie already.